src/style.css

```
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+
+  --main-color:rgba(0, 172, 220, 1);
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  right: 0;
+  align-items: center;
+ }
+
+ .logo {
```

Fix all cases (remove redundant whitespaces)
```suggestion
.logo {
```